### DISCUSSION TITLE
Show the time a package became deprecated

### DISCUSSION THREAD

**Discussion by noqcks** [npm/feedback#1065] [Registry]
https://registry.npmjs.com/request returns a `deprecated` key for a deprecated package version with an author message. I would assume npm keeps track of when an author deprecates a package version. 

Would it be possible to surface the datetime when the package was made deprecated?

Context: I'm the creator of https://github.com/xeol-io/xeol and we'd like to fetch data from NPM on when a package was deprecated.

**Comment by ljharb** [comment]
I think you can look at the "modified" time on the packument; it'll be later than "created" when it's been deprecated - or un-deprecated.

**Reply by noqcks** [reply]
This won't work because one may deprecate specific package versions

https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions#deprecating-a-single-version-of-a-package

and a specific package version from the API doesn't return a modified date

https://registry.npmjs.com/vue/2.7.4
